openLRSng/TX.h

```diff
@@ -696,7 +696,8 @@ static inline void processSpektrum(uint8_t c)
 
 static inline uint16_t SBUSToPpm(uint16_t input) {
   // Rescale [200, 1800] to [1000, 2000]
-  uint16_t value = (((input - 200) * 10) >> 4) + 1000;
+  // uint16_t value = (((input - 200) * 10) >> 4) + 1000;
```

Comment:
Probably remove the comment.

Reply:
ok, so you're ok with this change? you mentioned having to fix the scaling on the other end as well?

Reply:
Yes I am ok with it. If this is what's needed on the input side to create valid output, then that's what's needed.